.cirrus.yml

```
@@ -65,13 +65,15 @@ skip_task_on_pr: &SKIP_TASK_ON_PR
   # Skip this task on PRs if it does not have the fullci label,
   # it continues to run for direct pushes to master/release.
   skip: >
-    ($CIRRUS_PR != '' && $CIRRUS_PR_LABELS !=~ '.*fullci.*')
+    ! ( $CIRRUS_PR == '' || $CIRRUS_PR_LABELS =~ '.*fullci.*' )
 
 zam_skip_task_on_pr: &ZAM_SKIP_TASK_ON_PR
-  # Skip this task on PRs if it does not have the fullci or zamci label,
-  # it continues to run for direct pushes to master/release.
+  # Skip this task on PRs unless it has the fullci or zamci label
+  # or files in src/script_opt/** were modified.
+  # It continues to run for direct pushes to master/release, as
+  # CIRRUS_PR will not be empty.
```

```suggestion
  # CIRRUS_PR will be empty.
```

Alternatively we could rewrite this comment so it more directly mirrors the code, e.g.,

```
Skip this task unless we are running on a direct push to master/release (empty
`CIRRUS_PR`), are running on a PR labeled with `fullci` or `zamci`, or any
files in `src/script_opt/` were modified.
```